ksml-runner/src/test/java/io/axual/ksml/runner/config/KSMLRunnerConfigTest.java

```diff
@@ -0,0 +1,56 @@
+package io.axual.ksml.runner.config;
+
+/*-
+ * ========================LICENSE_START=================================
+ * KSML Runner
+ * %%
+ * Copyright (C) 2021 - 2022 Axual B.V.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class KSMLRunnerConfigTest {
```

Comment:
This class seems to be duplicate and fails the pipeline, should it be moved or renamed?

Reply:
Renamed the class to have uppercase KSML instead of Camel case. But for some reason the filename was not renamed in the same way. Corrected.